phpunit/functional/Document_ItemTest.php

```diff
@@ -76,7 +76,7 @@ public function testRelatedItemCloneRelations()
             }
 
             $item = \getItemForItemtype($itemtype);
-            $this->assertContains(Document_Item::class, $item->getCloneRelations(), $itemtype);
+            $this->assertContains(Document_Item::class, $item->getCloneRelations(), $itemtype, $itemtype);
```

Comment:
`assertContains` take 3 arguments maximum
```suggestion
            $this->assertContains(Document_Item::class, $item->getCloneRelations(), $itemtype);
```